src/features/product/apis/ProductsAPI.ts

```diff
@@ -0,0 +1,33 @@
+import { httpClient } from "../../../apis/httpClient";
+import { Product } from "../../../apis/types/response";
+import { CategoryOptionsKey, SortOptionsKey } from "../config/filter";
+
+const ERROR_MESSAGE = "상품 데이터를 가져오는 데 실패했습니다.";
+
+const ENDPOINT = "products";
+
+const sortOptionsMap: Record<SortOptionsKey, string> = {
+  "낮은 가격 순": "price,asc",
+  "높은 가격 순": "price,desc",
+} as const;
+
+export const ProductsAPI = {
+  get: async (
+    category: CategoryOptionsKey,
+    sortOption: SortOptionsKey
+  ): Promise<Product[]> => {
+    const params: Record<string, string> = { page: "0", size: "20" };
```

Comment:
```suggestion
    const params = { page: "0", size: "20" };
```
일러 땐 타입을 정의하지 않아도 된답니다 ㅎㅎ 오히려 정의하지 않았을 때 유추되는 타입이 더 정확해요.